docs/tutorials/first_steps/03_add_data.rst

```diff
@@ -144,3 +148,96 @@ Let us push the two fresh commits by running:
     Total 15 (delta 2), reused 0 (delta 0)
     To https://renkulab.io/gitlab/john.doe/flights-tutorial.git
         b55aea9..91b226b  master --> master
+
+
+Adding data from the local file system
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Often the data you wish to use on RENKU is not available on a remote repository.
```

Comment:
In the documentation, it is primarily written as `Renku` and not `RENKU`. We can discuss switching to `RENKU`, but I would make that in a separate PR.

Reply:
Sure, no worries.